packages/storybook/stories/va-table.stories.jsx

```diff
@@ -2,13 +2,14 @@ import React from 'react';
 import { getWebComponentDocs, propStructure, StoryDocs } from './wc-helpers';
 
 const tableDocs = getWebComponentDocs('va-table');
+const componentName = 'Table';
 
 export default {
-  title: 'Components/va-table',
+  title: `Components/${componentName}`,
   parameters: {
-    componentSubtitle: 'Table web component',
+    componentSubtitle: 'va-table',
     docs: {
-      page: () => <StoryDocs data={tableDocs} />,
+      page: () => <StoryDocs componentName={componentName} data={tableDocs} />,
```

Comment:
I don't think this `componentName` is really doing anything that the default doesn't do. Is there a reason we have it and others like it?

Reply:
Yes, this is how I am adding the sidebar maturity scale indicator for the web components.

I could just change the tag in the component-docs.json and it would accomplish the same thing, but I wasn't sure what else that would break.

Reply:
If we leave the object properties as they are in `additional-docs.js`, do we need to do this?